react/common/components/returnList/ListTableSchema.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import {
+  IconClock,
+  IconFailure,
+  IconVisibilityOn,
+  IconCheck,
+  IconSuccess,
+  IconExternalLinkMini,
+} from 'vtex.styleguide'
+import type { Status } from 'vtex.return-app'
+
+const status = {
+  new: 'new',
+  processing: 'processing',
+  picked: 'pickedUpFromClient',
+  pendingVerification: 'pendingVerification',
+  verified: 'packageVerified',
+  denied: 'denied',
+  refunded: 'amountRefunded',
+} as const
+
+/**
+ * @todo
+ * - Resolve messages
+ * - Resolve STATUS variable
+ */
+const ReturnListSchema = () => {
+  return {
+    properties: {
+      id: {
+        title: <FormattedMessage id="returns.requestId" />,
+        minWidth: 350,
+      },
+      sequenceNumber: {
+        title: <FormattedMessage id="returns.sequenceNumber" />,
+        width: 150,
+      },
+      orderId: {
+        title: <FormattedMessage id="returns.orderId" />,
+      },
+      createdIn: {
+        title: <FormattedMessage id="returns.submittedDate" />,
+        cellRenderer: ({ cellData }) => {
+          return new Date(cellData).toLocaleString()
+        },
+      },
+      status: {
+        title: <FormattedMessage id="returns.status" />,
+        // eslint-disable-next-line react/display-name
+        cellRenderer: ({ cellData }) => {
+          return <div>{renderIcon(cellData)}</div>
+        },
+      },
+    },
+  }
+}
+
+function renderIcon(requestStatus: Status) {
```

Comment:
I didn't check if this was already implemented. Maybe we should have this as 1 only source.

Reply:
Lets keep them independent; but what about in the same `utils` file?